includes/Core/Modules/Modules.php

```diff
@@ -968,6 +972,47 @@ private function get_rest_routes() {
 					),
 				)
 			),
+			new REST_Route(
```

Comment:
I'd suggest moving this route definition up a bit, so it's grouped with the other `core/modules`-prefixed routes.

Reply:
Sure thing!